client/src/components/RatingsAndReviews/dummy_data.js

```diff
@@ -0,0 +1,34 @@
+const data =
+{
+  "product": "63609",
+  "page": 0,
+  "count": 2,
+  "results": [
+    {
+      "review_id": 1115284,
+      "rating": 5,
+      "summary": "Great Product",
+      "recommend": true,
+      "response": null,
+      "body": "frad gthtwhr fahyaterf gtagf",
+      "date": "2021-12-22T00:00:00.000Z",
+      "reviewer_name": "tester",
+      "helpfulness": 1,
+      "photos": []
+    },
+    {
+      "review_id": 1115298,
+      "rating": 5,
+      "summary": "Great Product",
+      "recommend": true,
+      "response": null,
+      "body": "frad gthtwhr fahyaterf gtagf",
+      "date": "2021-12-22T00:00:00.000Z",
+      "reviewer_name": "tester",
+      "helpfulness": 1,
+      "photos": []
+    }
+  ]
+};
+
+export default data;
```

Comment:
Could you please add a new line at the end of the file? Just a tiny best practice thing.